api/controller/triggers.go

```diff
@@ -151,24 +151,57 @@ func SearchTriggers(database moira.Database, searcher moira.Searcher, page int64
 	return &triggersList, nil
 }
 
-func DeleteTriggersPager(dataBase moira.Database, pagerID string) (dto.TriggersSearchResultDeleteResponse, *api.ErrorResponse) {
-	exists, err := dataBase.IsTriggersSearchResultsExist(pagerID)
+func DeleteTriggersPager(database moira.Database, pagerID string) (dto.TriggersSearchResultDeleteResponse, *api.ErrorResponse) {
+	exists, err := database.IsTriggersSearchResultsExist(pagerID)
 	if err != nil {
 		return dto.TriggersSearchResultDeleteResponse{}, api.ErrorInternalServer(err)
 	}
 	if !exists {
 		return dto.TriggersSearchResultDeleteResponse{}, api.ErrorNotFound(fmt.Sprintf("pager with id %s not found", pagerID))
 	}
-	err = dataBase.DeleteTriggersSearchResults(pagerID)
+	err = database.DeleteTriggersSearchResults(pagerID)
 	if err != nil {
 		return dto.TriggersSearchResultDeleteResponse{}, api.ErrorInternalServer(err)
 	}
 	return dto.TriggersSearchResultDeleteResponse{PagerID: pagerID}, nil
 }
 
-func triggerExists(dataBase moira.Database, triggerID string) (bool, error) {
-	_, err := dataBase.GetTrigger(triggerID)
-	if err == database.ErrNil {
+// GetUnusedTriggerIDs returns unused triggers ids.
+func GetUnusedTriggerIDs(database moira.Database) (*dto.TriggersList, *api.ErrorResponse) {
+	triggerIDs, err := database.GetUnusedTriggerIDs()
+	if err != nil {
+		return nil, api.ErrorInternalServer(err)
+	}
+
+	triggerChecks, err := getTriggerChecks(database, triggerIDs)
+	if err != nil {
+		return nil, api.ErrorInternalServer(err)
+	}
+	triggersList := &dto.TriggersList{
+		List: triggerChecks,
+	}
+
+	return triggersList, nil
+}
+
+func getTriggerChecks(database moira.Database, triggerIDs []string) ([]moira.TriggerCheck, error) {
+	triggerChecks, err := database.GetTriggerChecks(triggerIDs)
+	if err != nil {
+		return nil, err
+	}
+	list := make([]moira.TriggerCheck, 0)
```

Comment:
Here, too, it's like it's better to add capacity

Reply:
fixed